python/ray/_private/event/export_event_logger.py

```diff
@@ -142,3 +142,29 @@ def get_export_event_logger(
             _export_event_logger[source_name] = ExportEventLoggerAdapter(source, logger)
 
         return _export_event_logger[source_name]
+
+
+def check_export_api_enabled(
+    source: ExportEvent.SourceType,
+) -> bool:
+    """
+    Check RAY_ENABLE_EXPORT_API_WRITE and RAY_ENABLE_EXPORT_API_WRITE_CONFIG environment
+    variables to verify if export events should be written for the given source type.
+
+    Args:
+        source: The source of the export event.
+    """
+    if ray_constants.RAY_ENABLE_EXPORT_API_WRITE:
+        return True
+    source_name = ExportEvent.SourceType.Name(source)
+    try:
+        ray_export_api_config_json = json.loads(
+            ray_constants.RAY_ENABLE_EXPORT_API_WRITE_CONFIG
+        )
```

Comment:
do we want to parse this every time? Or just once on ray launch / first time this config is read?

Reply:
Good point, updated to move the JSON parsing to where the env var is read